ceph/ceph.py

```diff
@@ -1928,7 +1928,7 @@ def container_prefix(self):
                 else ""
             )
 
-    def exec_command(self, cmd, **kw):
+    def exec_command(self, cmd, build="", **kw):
```

Comment:
Let's avoid changing the method signature unless it needs to be explicitly mentioned. We could always do `rhbuild = kw.get("build", None)` when the key needs to be retrieved.